devices/Hdc1080/version.json

```
@@ -0,0 +1,16 @@
+{
+  "$schema": "https://raw.githubusercontent.com/dotnet/Nerdbank.GitVersioning/master/src/NerdBank.GitVersioning/version.schema.json",
+  "version": "1.0.12-preview.{height}",
```

This should start from 0000 😉 
```suggestion
  "version": "1.0.0-preview.{height}",
```